igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/JenkinsConfig.groovy

```diff
@@ -40,11 +43,14 @@ import javax.validation.Valid
 @CompileStatic
 class JenkinsConfig {
 
+    @Value('${client.timeout:30}')
```

Comment:
Connection timeouts in other parts of Spinnaker are specified in
milliseconds.

Probably worth being consistent.
On Fri, Feb 5, 2016 at 10:40 AM Cameron Fieber <notifications@github.com>
wrote:

> In
> igor-web/src/main/groovy/com/netflix/spinnaker/igor/config/JenkinsConfig.groovy
> <https://github.com/spinnaker/igor/pull/63#discussion_r52054998>:
>
> > @@ -40,11 +43,14 @@ import javax.validation.Valid
> >  @CompileStatic
> >  class JenkinsConfig {
> >
> > +    @Value('${client.timeout:30}')
>
> I see below where it is explicitly seconds
>
> Maybe change the config property to client.readTimeoutSeconds
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spinnaker/igor/pull/63/files#r52054998>.
>